src/openapi/components/schemas/File.json

```diff
@@ -32,6 +32,10 @@
 			"$ref": "./S3Bucket.json",
 			"readOnly": true
 		},
+		"presignedPost": {
```

Comment:
@bickelj calling this out since I added it in as part of fixing the file documentation